client/TextArea.jsx

```diff
@@ -74,9 +74,10 @@ const TextArea = () => {
         />
         <div onChange={set("submission")}>
           <label htmlFor="submission">
-          <input aria-label="Offer" type="radio" name="submission" value="offer" defaultChecked={true}/> Offer
-          <input aria-label="Request" type='radio' name='submission' value='request'/> Request
-
+          <input id="offer" type="radio" name="submission" value="offer" defaultChecked={true}/>
+          <label for="offer" aria-label="offer">offer</label>
```

Comment:
question(nb): Do you need to use `htmlFor` here, since `for` might mean something else in the context of your JavaScript?

Reply:
Do you mean change for="xxx" to htmlfor="xxx"?